snuba/admin/package.json

```diff
@@ -29,5 +29,9 @@
     "ts-jest": "^29.0.5",
     "webpack": "^5.74.0",
     "webpack-cli": "^4.10.0"
+  },
+  "volta": {
+    "node": "19.9.0",
+    "yarn": "1.22.19"
```

Comment:
How are these versions chosen? Is that what we run in CI?

Reply:
no idea what is being run in CI, i presume a completely random version. I added the volta github action so this becomes the source of truth